Sources/PrimerSDK/Classes/User Interface/OAuth/OAuthViewController.swift

```diff
@@ -26,36 +26,30 @@ class OAuthViewController: UIViewController {
     }
 
     override func viewDidLoad() {
-        view.addSubview(indicator)
+        super.viewDidLoad()
         
+        let settings: PrimerSettingsProtocol = DependencyContainer.resolve()
```

Comment:
this file contains multiple view controllers. Since these contain a lot of crucial logic that we need to really test/ensure is stable, could we maybe split this up into two files?

Reply:
Will do. I believe however, that `OAuthViewController` only acts as a wrapper and could be completely removed. We would then just have the `WebViewController` initialized with different configs. I'll have to investigate a bit further the PayPal flow before doing that though.